packages/eslint-plugin/docs/rules/no-non-null-assertion.md

```diff
@@ -6,36 +6,39 @@ description: 'Disallow non-null assertions using the `!` postfix operator.'
 >
 > See **https://typescript-eslint.io/rules/no-non-null-assertion** for documentation.
 
-## Rule Details
+TypeScript's `!` non-null assertion operator asserts to the type system that an expression is non-nullable, as in not `null` or `undefined`.
+Using assertions to tell the type system new information is often a sign that code is not fully type-safe.
+It's generally better to structure program logic so that TypeScript understands when values may be nullable.
 
-Using non-null assertions cancels the benefits of the strict null-checking mode.
+## Examples
 
-Examples of code for this rule:
+<!-- Using non-null assertions cancels the benefits of the strict null-checking mode. -->
```

Comment:
is this commented out intentionally?

Reply:
...huh, I have no recollection of commenting this.

Reply:
was it supposed to be deleted?
It's text within the "examples" section.

Reply:
Oh, thank you, yes - that